### PR TITLE
unit test scripts

### PR DESCRIPTION
**Pull request by SAT-oO**

### CODE REVIEW

File: design/unit_testing/unit_test_inputs.m [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        else
+            val = currParam.range(1) + (currParam.range(2) - currParam.range(1)) .* rand(currParam.size);
+        end
+
+        testSet.(currParam.name) = val;
+    end
+end
+
+% models to run unit tests for
+models(1).name = 'AccelModel';
```
Comment (SAT-oO) [inline design/unit_testing/unit_test_inputs.m]: I would just use the `function_name` as the only source for reference

File: design/unit_testing/test_results.mat
Comment (SAT-oO) [inline design/unit_testing/test_results.mat]: the `results.mat` file only records one set of input and output for each test 
in reality, we will need different samples of input per test/function to verify that we are also handling the edge cases properly. 
you should make changes to the scripts so that we can change the `batch_size` parameter, and the script will just generate `n` set of inputs per batch

**Review by SAT-oO — COMMENTED** [review]
nice work

File: design/unit_testing/test_results.mat
Comment (michaely07) [reply design/unit_testing/test_results.mat]: added batch_size parameter so each time input generator runs it produces that number of inputs for each model

File: design/unit_testing/unit_test_outputs.m [hunk 2]
```diff
@@ -0,0 +1,23 @@
+load('pending_tests.mat');
+addpath(genpath('unit_testing'));
+addpath(genpath('model'));
```
Comment (SAT-oO) [inline design/unit_testing/unit_test_outputs.m]: rename this folder to sth like `automation_scripts` and move this `unit_testing` folder to be a subfolder of `unit-test` 
keep the file organization in a way that makes logical sense
